homework14/js/main.js

```diff
@@ -0,0 +1,154 @@
+function fieldNotEmpty(field){
+    if(field === '' || field === ' '){
+        alert('Field not be empty!');
+    }
+}
+
+//Return the century of the input year. The input will always be a 4 digit string, so there is no need for validation.
+const dateInput = document.querySelector('.first-block__input');
+const centuryButton = document.querySelector('.first-block__button');
+const resultCentury = document.querySelector('.first-block__result');
+
+function showCentury(stringOfYear){
+    if(stringOfYear === '' || stringOfYear === ' '){
+        alert('Field not be empty!');
+    } else
+    if(stringOfYear !== 4){
+        alert('Enter a year greater than 999')
+    } else
+    if(stringOfYear < 1100){
```

Comment:
Why not start from 1 year? 

```suggestion
    if(stringOfYear < 1){
```